parm/post_avblflds.xml

```diff
@@ -4282,9 +4282,9 @@
 
     <param>
        <post_avblfldidx>487</post_avblfldidx>
-       <shortname>GSD_PRES_ON_CLOUD_BASE</shortname>
-       <pname>PRES</pname>
-       <fixed_sfc1_type>cloud_base</fixed_sfc1_type>
+       <shortname>GSD_EXP_CEILING</shortname>
```

Comment:
@EdwardColon-NOAA The proposed changes in UPP ID 487 would have impact on 3drtma. Please let me know if you have concerns/suggestions.

Reply:
I will make this update in the python graphics scripts. I don't see any other issue.

Reply:
@EdwardColon-NOAA Thanks for clarifying for 3drtma. I will let you know when this PR is merged so you can update 3drtma's workflow.

Reply:
@EricJames-NOAA Could you update control file for rtma under parm/3drtma_postcntrl.xml as:
diff --git a/parm/3drtma_postcntrl.xml b/parm/3drtma_postcntrl.xml
index 7a2e662..23471ce 100755
--- a/parm/3drtma_postcntrl.xml
+++ b/parm/3drtma_postcntrl.xml
@@ -633,7 +633,7 @@
        </param>

        <param>
-           <shortname>GSD_PRES_ON_CLOUD_BASE</shortname>
+           <shortname>GSD_EXP_CEILING</shortname>
            <scale>3.0</scale>
        </param>

Then execute "make" to generating a new flat text file for rtma. Thanks!